src/FullTextSearchHelpers.ts

```diff
@@ -0,0 +1,118 @@
+﻿ /**
+ * FullTextSearchHelpers.ts
+ * Author: David de Regt
+ * Copyright: Microsoft 2017
+ *
+ * Reusable helper classes and functions for supporting Full Text Search.
+ */
+
+import _ = require('lodash');
+import SyncTasks = require('synctasks');
+
+import NoSqlProvider = require('./NoSqlProvider');
+import NoSqlProviderUtils = require('./NoSqlProviderUtils');
+
+const _charMap: { [char: string]: string } = {
```

Comment:
Why not _.deburr? It is different than this logic so perhaps you found this somewhere else?